File: problemas/matematica/2218/2218.c
#include <stdio.h>

int somaPA(int a1, int an, int n){
    return (n * (a1 + an)) / 2;
}

int main(){
    int t, n;

    scanf("%d\n", &t);

    for(int i = 0; i < t; ++i){
        scanf("%d\n", &n);
        printf("%d\n", somaPA(1, n, n) + 1);
    }

    return 0;
}

File: problemas/matematica/2218/2218.ml
let somaPA (a1 : int) (an : int) (n : int) : int = (n * (a1 + an)) / 2 in
Scanf.scanf "%d\n" (fun t ->
	for _ = 1 to t do
		Scanf.scanf "%d\n" (fun n ->
			Printf.printf "%d\n" ((somaPA 1 n n) + 1);
		)
	done;
)
